_data/menu.json

```diff
@@ -225,6 +225,10 @@
                     {
                       "page": "VALUES",
                       "url": "/docs/sql/query_syntax/values"
+                    },

```

Comment:
The rest of the menu items appear to be in alphabetical order?

Reply:
The ordering isn't quite alphabetical. It appears to be roughly in importance order (which is why I put FILTER at the bottom!). Do you mind if I leave it where it is?